client/src/components/Allocations/AllocationsModals/AllocationsTeamViewModal/AllocationsTeamViewModal.jsx

```diff
@@ -78,11 +78,20 @@ const AllocationsTeamViewModal = ({ isOpen, toggle }) => {
     },
     [dispatch]
   );
-
+  
   const [selectedTab, setSelectedTab] = useState(0);
   const handleTabChange = (e, newValue) => {
     setSelectedTab(newValue);
   };
+
+  let selectedUser = useState(null);
+  if (removingUserOperation) {
+    selectedUser = removingUserOperation.username;
+  }
+  if (card && card.username === selectedUser) {
+    setCard(null);
+  }
```

Comment:
The useState usage is incorrect. It should be
> const [selectedUser, setSelectedUser] = useState(null);

Otherwise selectedUser is a tuple of [null, setter]. 
When it is set in line 89, it simply overrides the value with username.

A reason all of this is still working is because in line 88, if condition is checking an object removingUserOperation, who value in cases where user is not removed will be
 ```
{
    "loading": false,
    "error": false,
    "username": ""
}
```
So, for all non-deletion scenarios, selectedUser is always empty string. 
And in line 91, the condition fails and setCard(null) will not execute.

In delete user scenario:
* removingUserOperation will be
```
{
    "loading": false,
    "error": false,
    "username": "foobar"
}
```
selectedUser will be "foobar"
line 91 check passes 
and setCard is null.

The test cases work, but the code needs cleanup:
line 87, useState is not needed, just initialize to null
line 88, check should be checking if removingUserOperation && removingUserOperation.username is not empty.
Rest of the code will work as is.